### ISSUE TITLE
Requesting new feature to enable NLU service.

### ISSUE THREAD

**Issue by nevrologica** [watson-developer-cloud/speech-javascript-sdk#157]
I have tried creating a browserified version of a set of service on using the node-sdk sources following the examples in the browserify folder but even though i managed to get a bundle it didn't work on the browser. The sources are missing various exports to method and none are exposed in order to use a namespace and call them properly. 

Plus that i would have to rewrite all the functionality this source has regarding the recognizeMicrophone, getVoices, getModels, getUserMedia method which i intent to keep for my current project as i have been working and building my application using them for quite a while now and it would be a pity to rewrite and test the whole lot.

So i made an effort to add NLU into this bundle and produce a minified version, that worked, but while the implementation for NLU using a get call with axios i get the infamous CORS header missing.

I can post the files i added, and it would be nice to see it included in the master head, however i need to resolve that pesky CORS warning. My request succeeds, i get the response, but for some strange reason i also get a warning. I can add the stuff on a new branch if i have permission to do so and it can then be reviewed or tested properly. I tried making a new pull request but i am not allowed to create a new branch.

**Comment by germanattanasio** [comment]
From what I know the service supports CORS but I would have to check. Can you open an issue in the `node-sdk` with what you did for the browser interaction and why it failed?

**Comment by nevrologica** [comment]
Researching futher i have been able to produce a working bundle, but i have been blocked by trying to make use of the generated accessToken i have. You can read more here: https://github.com/watson-developer-cloud/node-sdk/issues/1006 . However, i have been stack with the speech javascript custom sdk bundle that i included NLU to it, that works but with a hack in using a heroku proxy to aleviate the CORS warning. Not sure how to fix this, and not sure if i implement the NLU service correctly, but for i now i get results. Can i share the source code and how i am using it for suggestions?

**Comment by germanattanasio** [comment]
I think I'm going to close this as it's outside of the scope for this package. 

I will follow up on watson-developer-cloud/node-sdk#1006. Watson APIs should support CORS
